README.md

```diff
@@ -1,2 +1,6 @@
 # app
 URL fishing detection application containing frontend and service
+
+## Model service
+
+To configure the URL of the model service, make edit the environment.ts file and set modelServiceURL="YOUR_URL"
```

Comment:
Can you use backticks for better readability? e.g. `environments.ts`

Reply:
Ok, changed it. Please check again.